ts/input/tex/color/ColorMethods.ts

```diff
@@ -121,14 +121,13 @@ ColorMethods.DefineColor = function (parser: TexParser, name: string) {
  * @param {string} name The name of the control sequence.
  */
 ColorMethods.ColorBox = function (parser: TexParser, name: string) {
-  const cname = parser.GetArgument(name);
+  const model = parser.GetBrackets(name, '');
```

Comment:
Does `''` need to be `'named'`, like you do in `FColorBox` below?

Reply:
I'm OK either way (probably `''` is better as it is shorter), but would like to see the same treatment both here and at 142, whichever it is.

Reply:
Done.